### ISSUE TITLE
Training loss points in .tfevent differ from num_train_steps

### ISSUE THREAD

**Issue by DankoPenko** [AdeDZY/DeepCT#19]
Hi!

I would like to plot the training loss of the model, but in the .tfevents-file number of tags that begin with "loss_1" differ from the calculated number of steps ([num_train_steps](https://github.com/AdeDZY/DeepCT/blob/54fdaa530222dce6edbf349731aeea3944cf93ca/run_deepct.py#L1072))  .

As for example, for num_train_steps = 153596, my .tfevent-file has only 1552 tags of "loss_1".

Is there any hint of how to get the losses or how its being saved in .tfevent?

Thanks in advance!
